_rules/html-page-lang-valid-bf051a.md

```diff
@@ -63,31 +63,31 @@ The `lang` attribute specified is neither empty ("") nor only [ASCII whitespace]
 
 #### Failed Example 1
 
-The `lang` attribute value is not a valid primary language subtag.
+This `html` element has a `lang` attribute whose value is not a [valid language subtag][].
 
 ```html
 <html lang="xyz"></html>
 ```
 
 #### Failed Example 2
 
-The `lang` attribute value has a valid primary language subtag, but a syntactically invalid region subtag.
+This `html` element has a `lang` attribute whose value is not a [valid language subtag][]. Even though the [primary language subtag][] is valid, the [region subtag] is not
 
 ```html
 <html lang="en-US-GB"></html>
 ```
 
 #### Failed Example 3
```

Comment:
This failed example appears to be almost identical to failed example 1

Reply:
I disagree.
Failed Example 1 is about garbage lang attribute.
Failed Example 2 is about "the primary tag is not enough for this rule" (some other rules only check the primary tag).

Reply:
And failed examples 3 and 4? The description text is identical on both old and reworded versions.

I was comparing failed examples 1 with 3 and 4, but don't think my comment was clear about that. Apologies. Is it really necessary to have 3 versions on an invalid language tag?

Reply:
Another subject for a different PR!

Reply:
I agree that 1, 3 and 4 are very similar. Maybe not all are needed.
You've won #1097 😁